includes/class-wc-payment-token-swedbank-pay.php

```diff
@@ -120,14 +118,14 @@ public function set_recurrence_token( $masked_pan ) {
 	 */
 	public function is_default() {
 		// Mark Method as Checked on "Payment Change" page
-		if ( WC_Gateway_Swedbank_Pay_Cc::wcs_is_payment_change() &&
-		     isset( $_GET['change_payment_method'] ) &&
-			 abs( $_GET['change_payment_method'] ) > 0 )
-		{
+		if ( class_exists( 'WC_Subscriptions_Change_Payment_Gateway', false ) &&
+			 WC_Subscriptions_Change_Payment_Gateway::$is_request_to_change_payment &&
+			 isset( $_GET['change_payment_method'] ) &&
+			 abs( $_GET['change_payment_method'] ) > 0 ) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)